packages/esm-ugandaemr-app/src/pages/system-info/system-info.resource.tsx

```diff
@@ -0,0 +1,105 @@
+import useSWR from 'swr';
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { systemInfo } from './system-info.types';
+import { useState, useEffect } from 'react';
+
+export function useGetSystemInformation() {
+  const apiUrl = `/ws/rest/v1/systeminformation?v=full`;
+  const { data, error, isLoading } = useSWR<{ data: systemInfo }, Error>(apiUrl, openmrsFetch);
+
+  return {
+    systemInfo: data?.data,
+    isLoading,
+    isError: error,
+  };
+}
+
+export function useGetResourceInformation(type) {
+  const [state, setState] = useState({});
+  const [error, setError] = useState('');
+  const url = 'https://nhfr-staging-api.planetsystems.co/NHFRSearch?';
+  let param = '';
+
+  switch (type) {
+    case 'ownership':
+      param = 'resource=ValueSet&name=facilityOwnership';
+      break;
+    case 'careLevel':
+      param = 'resource=ValueSet&name=facilityLevel';
+      break;
+  }
+
+  useEffect(() => {
+    const dataFetch = async () => {
+      try {
+        const data = await (
+          await fetch(`${url}${param}&_pretty=true`, {
+            method: 'GET',
+            // headers: {
+            //   Authorization: `Basic ${btoa(`${username}:${password}`)}`,
+            //   'Content-Type': 'application/json',
+            // },
```

Comment:
let's also do some cleanup to get rid of commented code that won't be used

Reply:
Done the cleanup